### ISSUE TITLE
Add a tenant-aware request type for getting user credentials

### ISSUE THREAD

**Issue by Pickeringwareltd** [kreait/firebase-php#394]
**Is your feature request related to a problem? Please describe.**
I have a multi-tenanted authentication setup, I realise that this is handled within GIP however, there is very little support for it with PHP and the rest of my application uses firebase, as I'm sure other developers will too. 

**Describe the solution you'd like**
I'd like to add the tenantId to the appropriate requests, so that the tenant is taken into consideration when accessing user information, preventing a 'User not found error'. This is very easy to achieve: 

An example when getting user credentials via UID:
Add a nullable tenantID parameter to Firebase\Auth.php@getUser 
Add a nullable tenantID parameter to Firebase\Auth\ApiClient@getAccountInfo
Add the tenantId into the JSON request body like so:
```
return $this->requestApi('getAccountInfo', [
            'tenantId' => $tenantId,
            'localId' => $uid
        ]);
```

A null tenant ID defaults to the standard set of users, meaning this then allows tenant-aware clients to authenticate their users, whilst not affecting the current users who use it without multi-tenanted applications. 

**Describe alternatives you've considered**
I've looked around for a PHP SDK written for the  GIP but there is very little around, this seems to be the most concise platform for authentication.

**Comment by jeromegamez** [comment]
Thank you for this feature request, I‘ll look into it and hope that I will come up with a good implementation.

Please note that I will soon be dropping support for PHP <7.4 (probably after #390), so this feature will not be provided as a 4.x release.

Release 5.x will be fully compatible to 4.x though (except the PHP requirement and the removal of deprecated methods and classes), so if you consider using this SDK and are on a current PHP version, everything should be fine 😅

**Comment by martijnvdgrift** [comment]
Hey @jeromegamez ,

We're running into the same issue. We're storing our users in specific [Tenants](https://cloud.google.com/identity-platform/docs/multi-tenancy), but we're unable to delete this because this library isn't "tenant-aware". Trying to remove a user results in a `User not found` error.

Is this feature request still on your radar?

The "official" NodeJS library does have support for this, so maybe it can be used as inspiration.

Thanks!

**Comment by jeromegamez** [comment]
Hey @Pickeringwareltd and @martijnvdgrift - better late than never: Tenant awareness has landed with [release 5.12.0](https://github.com/kreait/firebase-php/releases/tag/5.12.0) 🥳🚀
